lte/gateway/c/session_manager/SessionManagement.md

```diff
@@ -0,0 +1,56 @@
+# Session Management in Magma
+
+## Overview of SessionD
+SessionD is the main service responsible for managing and enforcing session 
+configurations. 
+
+### Interfaces
+SessionD interfaces with three distinct external components. 
+1. Access (MME, AAA) <br>
+ This component receives all connection and session related information from 
+ the UE. It also handles mobility management. But from SessionD's point of view,
+ it notifies SessionD whenever a UE is attached or detached. This notification 
+ will include any context that SessionD needs for enforcement.
+2. Policy (PolicyDB, FeG+PCRF/OCS) <br>
+ This component is responsible for propagating any session or policy 
+ configuration that SessionD must enforce. Whenever a new UE is introduced, 
+ SessionD will reach out to this component to fetch any configuration. Depending
+ on the configuration, SessionD maybe reach out to this component for updates. 
+3. Switch (PipelineD) <br>
+ This component is responsible for policy and QoS enforcement. It receives 
+ any relevant policy and QoS configuration from SessionD and periodically 
+ reports state accordingly.
+ 
+### Internal Architecture 
+![](SessionD_Lg_Arch.png)
+
+## Configurations (Not Exhaustive)
+- FeG Relay <br>
+  Configured on the Orc8r by the `relay_enabled` flag. When it is enabled, it 
```

Comment:
maybe to be more specific we can say that this field can be found on NMS AGW network creation as `Network Type` on NMS. Or if you want the api reference point, through Federated LTE Networks `relay_enabled`

Reply:
When the network type is selected as federated_*, does it automatically set the LTE network config's relay_enabled field to true?

Reply:
Ohh you are right, i just checked on nms code and it only sets the FEG parameters. To do such thing, it would have to go to all the AGW and set it one by one.😓